truffle-box.json

```diff
@@ -1,6 +1,8 @@
 {
   "ignore": [
-    "README.md"
+    "README.md",
+    ".git",
+    ".gitignore"
```

Comment:
Maybe `.gitignore` should be available when unboxed? To avoid accidentally committing things like `.env`

Reply:
I left it in there because I saw @emilyJLin95 going in that direction. Maybe we should have a discussion about this behavior and apply it to all boxes. @emilyJLin95 thoughts?

Reply:
? Was I going in that directino? I think I specifically called out before that we should be keeping the `.gitignore`

Reply:
Looks like I zigged when I should have zagged! My bad. Would be happy to review a PR to fix it :) 

If `.gitignore` is to be added, I'd recommend not to  include the `.git` folder because there's a potential someone unboxes this into an existing git project.